pom.xml

```diff
@@ -51,7 +52,7 @@
         <maven.checkstyle.plugin.version>2.17</maven.checkstyle.plugin.version>
         <log4j.version>1.2.12</log4j.version>
         <!-- for Spring Boot, must be set to Hazelcast version shaded by hazelcast-jet -->
-        <hazelcast.version>3.10</hazelcast.version>
+        <hazelcast.version>3.10.3</hazelcast.version>
```

Comment:
this line should not be needed anymore

Reply:
I guess it is still used by docker compose, if not spring